spring/spring5/spring5-common/pom.xml

```diff
@@ -27,6 +27,10 @@
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-commons</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
```

Comment:
@tristantarrant sorry I've just seen this dependency here, I believed it was in another pom. The issue is that the common module is for both, embedded and remote. infinispan-core should not be integrated in the remote integration in any case, since this goes to the client code

Reply:
Anything needed here should go to commons, not to core

Reply:
I'm fixing the issue in another PR and reporting these changes in spring 6 branch